docs/reference/glossary.asciidoc

```diff
@@ -149,36 +188,52 @@ An index is a logical namespace which maps to one or more
 --
 // tag::index-alias-def[]
 // tag::index-alias-desc[]
-An index alias is a secondary name
-used to refer to one or more existing indices.
+An index alias is a logical name used to reference one or more indices.
 
-Most {es} APIs accept an index alias
-in place of an index name.
+Most {es} APIs accept an index alias in place of an index name.
 // end::index-alias-desc[]
 
 See {ref}/indices-add-alias.html[Add index alias].
 // end::index-alias-def[]
 --
 
+[[glossary-index-lifecycle]] index lifecycle ::
+// tag::index-lifecycle-def[]
+The four phases an index can move through as it ages: <<glossary-hot-phase>>, <<glossary-warm-phase>>, 
```

Comment:
Looks like line wrapping may be off here.

Reply:
Was trying to get at the fact that a lifecycle inherently has a time component, but given that the main point of ILM is to be able to use criteria other than age to move through the lifecycle, it's confusing.